core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -103,6 +103,7 @@ class Application < ::Rails::Application
 Spree.config do |config|
   config.mails_from = "store@example.com"
   config.raise_with_invalid_currency = false
+  config.use_combined_first_and_last_name_in_address = true
```

Comment:
This should have been `false`, because we still tested v2.11 here, that has this set to `false` (see above)
That way specs are misleadingly green and issues like #4094 didn't came up in the specs for v2.11

I am proposing to revert this in v2.11 branch